libs/etterlatte-behandling-model/src/main/kotlin/no/nav/etterlatte/libs/common/behandling/BehandlingStatus.kt

```diff
@@ -58,7 +58,6 @@ enum class BehandlingStatus {
                 ATTESTERT,
                 OPPRETTET,
                 VILKAARSVURDERT,
-                TRYGDETID_OPPDATERT,
```

Comment:
Litt usikker på om jeg forsto denne? Skal nå TRYGDETID_OPPDATERT omregnes ved g-regulering også?

Reply:
https://nav-it.slack.com/archives/C06G4NDTD0U/p1716470713294059

Reply:
Det kan ha blitt lagt til beløper i overstyrt beregning med status trygdetid_oppdatert. Og sjekken på om det finnes åpen behandling for en sak med overstyrt beregning baseres på tilbakestilteBehandlinger. Og når TRYGDETID_OPPDATERT ikke blir tilbakestilt vil den heller ikke feile selv om det er en åpen behandling.

Vanskelig å forklare enkelt i tekst. Er du veldig nysgjerrig kan vi ta det på Zoom.